sdk/devcenter/Azure.Developer.DevCenter/src/Generated/Models/OSDisk.cs

```diff
@@ -46,7 +46,7 @@ public partial class OSDisk
         private IDictionary<string, BinaryData> _serializedAdditionalRawData;
 
         /// <summary> Initializes a new instance of <see cref="OSDisk"/>. </summary>
-        internal OSDisk()
+        public OSDisk()
```

Comment:
we need to figure out the root cause of this change, and make sure our new code is correct.
Otherwise this change should be undone

Reply:
The change is because the usage of `OSDisk` change to `input,output,json` ( previous is `output,json`). The latest tcgc will propagate usage override to members, and the root model `DevBox` has a usage override to `input,output`, so now all his included model will change the usage to include `input, output`.
https://github.com/Azure/azure-rest-api-specs/blob/63dc797c014849ef0c5da7eb91197975efb6a474/specification/devcenter/DevCenter/client.tsp#L69